apps/site/src/views/Home/components/Title/Title.module.scss

```diff
@@ -0,0 +1,6 @@
+@use "bootstrap-utils" as utils;
+
+.title {
+	width: clamp(18rem, 60vw, 70rem);
+	height: clamp(5rem, 10vw, 10rem);
+}
```

Comment:
Suggestion: might want to add some padding to the title or make it smaller because it's gotten a lot bigger.

<img width="960" alt="image" src="https://github.com/HackAtUCI/zothacks-site-2023/assets/28026520/cac07455-b11e-4b80-8190-1c3e58b62b8e">

Reply:
Still looking a bit comically large: the height should probably not exceed our`$h1-font-size` by much.

Reply:
@samderanova I tried padding but it didn't seem to work, so I used clamp instead.

Reply:
@taesungh I think it is good enough for now. Feel free to fix this yourself or have someone address it in another PR.